bioimageio/spec/v0_3/raw_nodes.py

```diff
@@ -33,7 +33,7 @@
 PostprocessingName = Literal[
     "binarize", "clip", "scale_linear", "sigmoid", "zero_mean_unit_variance", "scale_range", "scale_mean_variance"
 ]
-Type = Literal["model", "dataset", "application", "notebook"]
+Type = NewType("Type", str)
```

Comment:
```suggestion
Type = str
```

As we do not create every type value by calling `Type("type string")`